src/python/impactx/dashboard/requirements.txt

```diff
@@ -0,0 +1,45 @@
+aiohttp==3.9.5
+aiosignal==1.3.1
+altair==5.3.0
+async-timeout==4.0.3
+attrs==23.2.0
+contourpy==1.2.1
+cycler==0.12.1
+fonttools==4.53.0
+frozenlist==1.4.1
+idna==3.7
+Jinja2==3.1.4
+jsonschema==4.22.0
+jsonschema-specifications==2023.12.1
+kiwisolver==1.4.5
+MarkupSafe==2.1.5
+matplotlib==3.9.0
+more-itertools==10.2.0
+msgpack==1.0.8
+multidict==6.0.5
+numpy==1.26.4
+packaging==24.0
+pandas==2.2.2
+pillow==10.3.0
+pyparsing==3.1.2
+python-dateutil==2.9.0.post0
+pytz==2024.1
+referencing==0.35.1
+rpds-py==0.18.1
+six==1.16.0
+tenacity==8.3.0
+toolz==0.12.1
+trame==3.6.2
```

Comment:
Please reduce the entries here to direct dependencies we need to install, e.g., `trame` and other packages you directly use, and their minimal versions.

Transitive dependencies are maintained by our dependencies, let us not list them here for maintainability.

Reply:
Should the version be included as well?

Reply:
Minimal version would be good, yes